plugins/node/opentelemetry-plugin-dns/test/integrations/dnspromise-lookup.test.ts

```diff
@@ -190,7 +190,7 @@ describe('dns.promises.lookup()', () => {
             // tslint:disable-next-line:no-any
             hostname: hostname as any,
             forceStatus: {
-              code: CanonicalCode.INVALID_ARGUMENT,
+              code: StatusCode.ERROR,
```

Comment:
After the changes in StatusCode, both cases of the `if` are the same, so it can be collapsed into a single case and the if can be removed